content/number-theory/ModularArithmetic.h

```diff
@@ -3,14 +3,14 @@
  * Date: 2009-09-28
  * License: CC0
  * Source: folklore
- * Description: Operators for modular arithmetic. You need to set {\tt mod} to
- * some number first and then you can use the structure.
+ * Description: Operators for modular arithmetic. Update mod.
+ * Use commented code in invert if mod is not prime.
  */
 #pragma once
 
 #include "euclid.h"
 
-const ll mod = 17; // change to something else
+const ll mod = 17;
```

Comment:
Yo dejaría el `// change to something else`, ya lo dice en la descripción, pero de cualquier manera no hace que aumente la cantidad de lineas